src/containerapp/azext_containerapp/_clients.py

```diff
@@ -20,13 +20,14 @@
     ContainerAppsJobClient,
     DaprComponentClient,
     ManagedEnvironmentClient,
-    WorkloadProfileClient)
+    WorkloadProfileClient,
+    StorageClient)
 
 from knack.log import get_logger
 
 logger = get_logger(__name__)
 
-PREVIEW_API_VERSION = "2023-08-01-preview"
+PREVIEW_API_VERSION = "2023-11-02-preview"
```

Comment:
You should not change api version for all operations because manifest has not been rollout. Or we need to make sure the commands not be released until manifest rollout complete.